fabric/Makefile

```diff
@@ -90,9 +90,9 @@ $(FABRIC_PEER_BIN): $(TLCC_GO_SRC) $(TLCC_ENCLAVE_LIB) $(FABRIC_PATCHED)
 	   GO_TAGS=FPC \
         $(MAKE) peer
 
-native: peer
+native:
 	cd $(FABRIC_PATH) && \
-	GOFLAGS="-mod=mod" $(MAKE) -j orderer cryptogen configtxgen
+	GOFLAGS="-mod=mod" $(MAKE) -j orderer cryptogen configtxgen peer
```

Comment:
Are you sure we do not need the proxy patch anymore? I think it is still needed for the integration test which mixes fpc with standard go chaincode, the latter being directly run by peer (at least if you don't yet have the corresponding docker image i think you could run into trouble behind a proxy)?  In that case, might be easier to leave Makefile as-is and just `git mv` every patch except proxy to `fabric/patches.disabled/`?

Reply:
Good point, I have not run it in HW mode.
* For IAS, we don't need it anymore. Since that part has moved to the client side.

* For standard chaincode, I guess that would allow us to install any package in the container. However, the solution is pre-external-builder. Should we move to an external builder for regular chaincodes behind proxy?

Reply:
i don't think it's worth to go to the effort of doing so. Given that for integration tests we anyway compile our own peer, i wouldn't worry about that patch added. Certainly the least effort for us (by leaving makefile as is and a simple git mv (or git rm) to (re)move the non-wanted patches?

Reply:
sounds good, though for development in this repo.
In the future we should really move to the external builder.

Reply:
fabric is not moving to external builder by default for legacy chaincode, so why should we spend effort on it?

Reply:
Because, before, Fabric lacked a way to deal with proxies, now it does not.

Alternatively, we could try to push the patch to Fabric.

Reply:
still seems to me rather low on the priority list, at least as long as the current patch works for us as-is? (i definitely would agree though that it's not worth spending any time maintaining/updating the patch)